### DISCUSSION TITLE
Downloads Media

### DISCUSSION THREAD

**Discussion by Lotsoo** [robertodoering/harpy#608] [Ideas]
I have an idea that is when you download an image or video it is saved directly to the gallery or a draft image.  So when I want to post to social media, I don't have to go to the file manager first.
![Screenshot_2022-03-19-21-55-42-794_com robertodoering harpy free](https://user-images.githubusercontent.com/65107724/159126504-3b49fdea-b2c8-4ce2-887f-546515e6ab1d.jpg)

**Comment by robertodoering** [comment]
This is coming with the next update
